logstash-core/lib/logstash/api/lib/app.rb

```diff
@@ -25,6 +25,19 @@ def initialize(app=nil)
       @factory = CommandFactory.new(settings.service)
     end
 
+    get "/_directory" do
```

Comment:
Is there any prior art for the name of this route? Maybe if we look at hypermedia type of api?